snuba/api.py

```diff
@@ -262,25 +269,32 @@ def json_default(obj):
 
 
 @split_query
-def parse_and_run_query(dataset, request: Request, timer):
-    from_date, to_date = TimeSeriesExtensionProcessor.get_time_limit(request.extensions['timeseries'])
+def parse_and_run_query(
+    dataset,
+    request: Request,
+    extension_payloads: Mapping[str, Mapping[str, Any]],
+    timer,
+):
+    from_date, to_date = TimeSeriesExtensionProcessor.get_time_limit(
+        TimeSeriesExtension.parse_payload(extension_payloads['timeseries'])
+    )
```

Comment:
I think we should go ahead and remove this now.

Reply:
I just noticed these fields are actually stored in the stats dict but it is never used.
Do you have any context on this ?

Reply:
These are published to Kafka (used only for manual tailing right now) and Redis (used on the dashboard, which is mysteriously broken.) I don't think it's worth keeping around, at least in its current implementation. It might make sense later for the extension processor to be able to record some stats at some point, but I don't know if that's a problem that needs to be solved right now.